notebooks/py_scripts/03-Process.py

```diff
@@ -207,26 +211,54 @@
 # %%
 model.ModelEvaluation.heading
 
+# %% [markdown]
+# If your project was initialized in a version of DeepLabCut other than the one you're currently using, model evaluation may report key errors. Specifically, your `config.yaml` may not specify `multianimalproject: false`.
+
 # %%
 model.ModelEvaluation.populate()
 
 # %%
 model.ModelEvaluation()
 
 # %% [markdown]
-# ## Pose Estimation
+# ### Pose Estimation
 
 # %% [markdown]
-# To put this model to use, we'll conduct pose estimation on the video generated in the [DataDownload notebook](./00_DataDownload_Optional.ipynb). Here, we can also specify parameters accepted by the `analyze_videos` function as a dictionary.
+# To put this model to use, we'll conduct pose estimation on the video generated in the [DataDownload notebook](./00_DataDownload_Optional.ipynb). First, we need to update the `VideoRecording` table with the recording from a session.
+
+# %%
+key = {'subject': 'subject6',
+       'session_datetime': '2021-06-02 14:04:22',
+       'recording_id': '1', 'camera_id': 1}
+model.VideoRecording.insert1(key)
+                         # do not include an initial `/` in relative file paths   
+key.update({'file_id': 1, 
+            'file_path': 'openfield-Pranav-2018-10-30/videos/m3v1mp4-copy.mp4'})
+model.VideoRecording.File.insert1(key, ignore_extra_fields=True)
```

Comment:
Since this is a didactic notebook, I would suggest only inserting the attributes specific to this table.